packages/admin-ui/src/combobox/combobox-popover-base.tsx

```diff
@@ -12,102 +12,113 @@ import { Spinner } from '../spinner'
 import { Button } from '../button'
 import { useMessageFormatter } from '../i18n'
 import type { ComboboxMultipleState } from '.'
+import { cx } from '@vtex/admin-ui-core'
 
-export const ComboboxPopoverBase = createComponent<
-  typeof AriakitComboboxPopover,
-  Props
->((props) => {
-  const { state, onRetry = () => null, children, ...restProps } = props
+export const ComboboxPopoverBase = forwardRef(
+  (props: ComboboxPopoverBaseProps, ref: Ref<HTMLDivElement>) => {
+    const {
+      state,
+      onRetry = () => null,
+      children,
+      className = '',
+      ...htmlProps
+    } = props
 
-  const { deferredValue, status, ...comboboxState } = state
+    const { deferredValue, status, ...comboboxState } = state
 
-  const formatMessage = useMessageFormatter(messages.popover)
+    const formatMessage = useMessageFormatter(messages.popover)
 
-  const renderChildren = () => {
-    switch (status) {
-      case 'ready': {
-        return <>{children}</>
-      }
+    const renderChildren = () => {
+      switch (status) {
+        case 'ready': {
+          return <>{children}</>
+        }
 
-      case 'loading': {
-        return (
-          <Box
-            csx={{
-              margin: '$space-2',
-            }}
-          >
-            <Spinner />
-          </Box>
-        )
-      }
+        case 'loading': {
+          return (
+            <Box
+              csx={{
+                margin: '$space-2',
+              }}
+            >
```

Comment:
You can apply this change in the other components using `csx=`
```suggestion
            <div
              className={csx({
                margin: '$space-2',
              })}
            >
```

Reply:
Maybe you can also create a constant for this value since it's used in several places
```suggestion
            <div className={popoverStatusContainerTheme} >
```